sdk.mjs

```diff
@@ -136,7 +136,7 @@ export const getIconDataPath = (
 /**
  * Get contents of *_data/simple-icons.json*.
  * @param {String} rootDir Path to the root directory of the project
- * @returns {String} Content of *_data/simple-icons.json*
+ * @returns {Promise<String>} Content of *_data/simple-icons.json*
```

Comment:
Fine but I'm rewriting this on #10276, it is considered a breaking change even when is a bug in types so revert it when you finish, please.